reconcile/quay_mirror.py

```diff
@@ -147,15 +151,21 @@ def process_repos_query(
                 server_url = quay_repo["org"]["instance"]["url"]
 
                 for item in quay_repo["items"]:
-                    if images and item["name"] not in images:
+                    if item["mirror"] is None:
                         continue
 
-                    if item["mirror"] is None:
+                    mirror_url = item["mirror"]["url"]
+
+                    if repository_urls and mirror_url not in repository_urls:
```

Comment:
nit: depending how often we run the `for item in ...` loop, it might be faster to have `repository_urls` defined as a `set` - just a nit, ignore if you want

Reply:
makes sense. Added in ce7404cd6f0ae484d514d69ad67d452259618cdc